packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- `CustomSelectControlV2`: Remove "experimental" designation ([#61078](https://github.com/WordPress/gutenberg/pull/61078)).
+
```

Comment:
I guess we could remove the changelog entry for this one, since this is a housekeeping change that doesn't really affect anyone.